provider/testing/resource.go

```diff
@@ -32,8 +32,11 @@ type ResourceTestCase struct {
 	ParallelFetchingLimit uint64
 	// SkipIgnoreInTest flag which detects if schema.Table or schema.Column should be ignored
 	SkipIgnoreInTest bool
+	Verifiers        []Verifier
 }
 
+type Verifier func(t *testing.T, conn pgxscan.Querier)
```

Comment:
would maybe make the verifier signature like this, and the default verifier will be no empty columns
```suggestion
type Verifier func(t *testing.T, table *schema.Table, conn pgxscan.Querier, SkipIgnoreInTest bool)
```

Reply:
To support ignoring flag, I changed signature as you suggested. Now single verifier will be used to verify one or more tables (main table and relations) from specific *schema.Table. So in ResourceTestCase's Verifiers field we have map from resource name to list of verifiers for this resource.